yarn.lock

```diff
@@ -8956,11 +8956,6 @@ jest-message-util@^23.4.0:
     slash "^1.0.0"
     stack-utils "^1.0.1"
 
-jest-mock-console@^1.0.0:
```

Comment:
Has this been removed? Was it not needed?

Reply:
Hmm @foxleigh81,  I didn't consciously remove it.  Hmm, I can't remember if we're meant to commit `yarn.lock` files or not. I think I read conflicting opinions.

Reply:
I suspect that you may have got an outdated yarn.lock file then, perhaps you accidentally did `npm something` rather than `yarn something` at some point? Or for some reason when you update your dependencies, the `yarn.lock` just didn't update. This might potentially be causing `yarn test` to fail if the CI tool doesn't have `jest-mock-console` installed. I'd expect this to still install it though provided it's in package.json so hopefully, it's not an issue.